thrd_rcu/Makefile

```diff
@@ -0,0 +1,19 @@
+CFLAGS = -Wall
+CFLAGS += -g
+CFLAGS += -D'READER_NUM=10'
```

Comment:
Change to `N_READERS`. However, for RCU, the number of writer should be always 1.